ouroboros-consensus/src/Ouroboros/Consensus/Storage/ImmutableDB/API.hs

```diff
@@ -303,16 +303,16 @@ instance Ord (CompareTip blk) where
 -------------------------------------------------------------------------------}
 
 -- | Errors that might arise when working with this database.
-data ImmutableDBError
-  = ApiMisuse ApiMisuse PrettyCallStack
+data ImmutableDBError blk
```

Comment:
Remove the `ExistentialQuantification` pragma from this file, as it should not be needed anymore.